Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -88,8 +93,13 @@ o2_add_dpl_workflow(hf-task-lc
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing
                     COMPONENT_NAME Analysis)
 
-o2_add_dpl_workflow(hf-task-jpsi
-                    SOURCES taskJpsi.cxx
+o2_add_dpl_workflow(hf-task-jpsi-toee
+                    SOURCES taskJpsiToEE.cxx
```

Comment:
@ginnocen Didn't we want to have one task per hadron, that would include all considered decay channels?

Reply:
in principle yes. After your PID refactory we should be able to merge everything together.